pkg/apis/pingcap/v1alpha1/types.go

```diff
@@ -2020,12 +2065,18 @@ type RestoreSpec struct {
 	To *TiDBAccessConfig `json:"to,omitempty"`
 	// Type is the backup type for tidb cluster.
 	Type BackupType `json:"backupType,omitempty"`
```

Comment:
Can be update comment to what it exactly means?

Reply:
ok